pkg/apis/crd/v1alpha3/types.go

```diff
@@ -99,3 +99,27 @@ type ClusterGroupList struct {
 
 	Items []ClusterGroup `json:"items,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type Group struct {
```

Comment:
Add some comments to say Group can be used in Antrea NetworkPolicy, and when used for AppliedTo it cannot include a Namespace selector.

@Dyanngg

Reply:
Agreed